challenge_30_days/day_8.py

```diff
@@ -10,4 +10,22 @@
 
     Give me a number: 125
     No. given number is not palindrome number
-'''
\ No newline at end of file
+'''
+
+def check_palindrome_number(number):
+    return number == number[::-1] 
+    # This will print the string backwords and see if it stays the same as the original. 
```

Comment:
This does not print anything. This is a condition, if number == number (backwards) it is true

Reply:
OK